File: Java/Assignment3_1.java

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// file name   : Assignment3_1.java
// Description : write a program which accept one number from user and print that number of even numbers on screen.
// Author      : Aniket Devaba Markad                        
// Date        : 28/11/2025 
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class number
{

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Function Name : PrintEven 
// Description   : This function is used to print even numbers. 
// Author        : Aniket Devaba Markad 
// Date          : 28/11/2025 
// 
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

    void PrintEven(int iNo)
    {
        int iCnt = 0,iEven = 0;

        if(iNo < 0)
        {
            iNo = -iNo;
        }
        
        for(iCnt = 1;iCnt <= iNo; iCnt++)
        {
            System.out.print(iEven+" ");
            iEven += 2;
        }
        System.out.println();
    }
}

public class Assignment3_1
{
    public static void main(String a[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.println("Enter the number");
        iValue = sobj.nextInt();

        number nobj = new number();
        nobj.PrintEven(iValue);
    }
}

File: CPP/Assignment3_1.cpp
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// file name   : Assignment3_1.cpp
// Description : Write a program which accept one number from user and print that number of even numbers on screen.
// Author      : Aniket Devaba Markad
// Date        : 09/12/2025
//
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<iostream>
using namespace std;

//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Function Name : PrintEven
// Description   : This function is used to print even numbers.
// Input         : int
// Output        : void
// Author        : Aniket Devaba Markad
// Date          : 09/12/2025
//
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

void PrintEven(int iNo)
{
    int iCount = 0;
    int iCnt = 0;

    if(iNo < 0)
    {
        return;
    }

    iCnt = 1;
    while(iCount < iNo)
    {
        if(iCnt % 2 == 0)
        {
            cout << iCnt << "\n";
            iCount++;
        }
        iCnt++;
    }
}

int main()
{
    int iValue = 0;

    cout << "Enter number:";
    cin >> iValue;

    PrintEven(iValue);

    return 0;
}
